pkg/dashboard/apiserver/event/event.go

```diff
@@ -77,10 +80,13 @@ func (s *Service) list(c *gin.Context) {
 		log.V(1).Info("Replace query namespace with", ns)
 	}
 
+	start, _ := time.Parse(time.RFC3339, c.Query("start"))
+	end, _ := time.Parse(time.RFC3339, c.Query("end"))
```

Comment:
I think it's better to judge the error, otherwise, an invalid request can make the dashboard panic.

Reply:
Agreed! I will update soon.

Reply:
Done in ddd08ef.

Reply:
It looks like parse will always return zero time if the second param is incorrect. 🔮 I will reset the previous commit.